pkgs/by-name/ka/kaufkauflist/package.nix

```diff
@@ -1,37 +1,44 @@
-{ lib
-, buildPackages
-, fetchFromGitHub
-, buildNpmPackage
-, fetchFromGitea
-, nix-update-script
+{
+  lib,
+  buildPackages,
+  fetchFromGitHub,
+  buildNpmPackage,
+  fetchFromGitea,
+  nix-update-script,
 }:
 
 let
   esbuild' = buildPackages.esbuild.override {
-    buildGoModule = args: buildPackages.buildGoModule (args // rec {
-      version = "0.19.11";
-      src = fetchFromGitHub {
-        owner = "evanw";
-        repo = "esbuild";
-        rev = "v${version}";
-        hash = "sha256-NUwjzOpHA0Ijuh0E69KXx8YVS5GTnKmob9HepqugbIU=";
-      };
-      vendorHash = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
-    });
+    buildGoModule =
+      args:
+      buildPackages.buildGoModule (
+        args
+        // rec {
+          version = "0.19.11";
+          src = fetchFromGitHub {
+            owner = "evanw";
+            repo = "esbuild";
+            rev = "v${version}";
+            hash = "sha256-NUwjzOpHA0Ijuh0E69KXx8YVS5GTnKmob9HepqugbIU=";
+          };
+          vendorHash = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
+        }
+      );
```

Comment:
> error: builder for '/nix/store/qsnj6jw4fqbi2fdg7gkbqmclbdvv3vba-kaufkauflist-4.0.2.drv' failed with exit code 1;
       last 10 log lines:
       > npm error     throw new Error(`Expected ${JSON.stringify(versionFromPackageJSON)} but got ${JSON.stringify(stdout)}`);
       > npm error           ^
       > npm error
       > npm error Error: Expected "0.21.5" but got "0.19.11"
       > npm error     at validateBinaryVersion (/private/tmp/nix-build-kaufkauflist-4.0.2.drv-0/source/node_modules/esbuild/install.js:133:11)
       > npm error     at /private/tmp/nix-build-kaufkauflist-4.0.2.drv-0/source/node_modules/esbuild/install.js:281:5
       > npm error
       > npm error Node.js v20.18.0
       > npm error Log files were not written due to an error writing to the directory: /nix/store/qicd58ma2s73wpkg1yicvnqqqfgfgw80-kaufkauflist-4.0.2-npm-deps/_logs
       > npm error You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
       For full logs, run 'nix-store -l /nix/store/qsnj6jw4fqbi2fdg7gkbqmclbdvv3vba-kaufkauflist-4.0.2.drv'.
error: 1 dependencies of derivation '/nix/store/jwjv00kr9px7lnrnx16sysidbvsnbfbk-review-shell.drv' failed to build

It may need to be built with the latest esbuild.

Reply:
well 0.21.5 is not the latest but it doesn't want the latest (0.24.0)

Reply:
okie, it's updated now to 0.21.5